packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -548,6 +662,22 @@ class ScrollableBaseNative extends Component {
 		}
 	}
 
+	initHorizontalOverscrollRef = (ref) => {
+		if (ref) {
+			this.overscrollRefs['horizontal'] = ReactDOM.findDOMNode(ref); // eslint-disable-line react/no-find-dom-node
+			this.newOverscrollJob('horizontal', 'before');
+			this.newOverscrollJob('horizontal', 'after');
+		}
+	}
+
+	initVerticalOverscrollRef = (ref) => {
+		if (ref) {
+			this.overscrollRefs['vertical'] = ref;
```

Comment:
Did you intended to use `ref` instead of `ReactDOM.findDOMNode(ref)`?

Reply:
`ref` is not a React component but a `div` node actually here. So, yes, we don't need `findDOMnode`.